### PR TITLE
feat: set webpack mode to development if missing

### LINKED ISSUE #86: Set mode: development
When using Webpack it is recommended to set `mode` which allows plugins and presets to be set correctly.

https://webpack.js.org/configuration/mode/

We should set `mode: development` by default if not specified

### PR DESCRIPTION
**Pull request by bahmutov**
- closes #86

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +95,8 @@ Object of webpack options. Just `require` in the options from your `webpack.conf
 
 Source maps are always enabled unless explicitly disabled by specifying `devtool: false`.
```
Comment (Bkucera) [inline README.md:96]: I believe this is correct
```suggestion
Source maps are always enabled unless explicitly disabled by specifying `devtool: false` or `mode: "production"`
```

File: README.md [hunk 2]
```diff
@@ -94,6 +95,8 @@ Object of webpack options. Just `require` in the options from your `webpack.conf
...
 
+Webpack [mode](https://webpack.js.org/configuration/mode/) is set to `development` if not present. You can set `mode` to "development, "production" or "none".
```
Comment (Bkucera) [inline README.md:98]:
```suggestion
Webpack [mode](https://webpack.js.org/configuration/mode/) is set to `development` if not present. You can set `mode` to "development", "production" or "none".
```

File: README.md (hunk 1 above)
Comment (bahmutov) [reply README.md:96]: probably yes

Comment (brian-mann) [reply README.md]: Nope, we are always setting `devtool` even if `mode` is set to `production`. I believe we should **always** set `devtool` unless its explicitly turned off. The code is correct, the `README` is wrong.

**Pull request merged**